src/mpid/ch4/netmod/include/netmod.h

```diff
@@ -73,6 +73,7 @@ typedef int (*MPIDI_NM_rma_win_cmpl_hook_t) (MPIR_Win * win);
 typedef int (*MPIDI_NM_rma_win_local_cmpl_hook_t) (MPIR_Win * win);
 typedef int (*MPIDI_NM_rma_target_cmpl_hook_t) (int rank, MPIR_Win * win);
 typedef int (*MPIDI_NM_rma_target_local_cmpl_hook_t) (int rank, MPIR_Win * win);
+typedef bool(*MPIDI_NM_rma_am_progress_cond_check_t) (MPIR_Win * win);
```

Comment:
How about name the function more explicitly, e.g. `MPIDI_NM_win_need_am_progress(win)`?